### ISSUE TITLE
Arguments required for running  train_gpt2_summarizer.py 

### ISSUE THREAD

**Issue by SumanthSrungavarapu** [SKRohit/Generating_Text_Summary_With_GPT2#10]
I am trying to recreate what you have done and getting this error.
Can you provide a optimal set of parameters

(tensorflow) C:\Users\XXXX\Desktop\Generating_Text_Summary_With_GPT2-master>python train_gpt2_summarizer.py --batch_size 1 --root_dir CNN\gpt2_1024_data
usage: train_gpt2_summarizer.py [-h] --lr LR [--seed SEED] [--n_gpu N_GPU] --gradient_accumulation_steps
                                GRADIENT_ACCUMULATION_STEPS --batch_size BATCH_SIZE [--num_workers NUM_WORKERS]
                                [--device DEVICE] --num_train_epochs NUM_TRAIN_EPOCHS --output_dir OUTPUT_DIR
                                --model_dir MODEL_DIR [--fp16 FP16] [--fp16_opt_level FP16_OPT_LEVEL]
                                [--max_grad_norm MAX_GRAD_NORM] [--root_dir ROOT_DIR] [--ids_file IDS_FILE]
train_gpt2_summarizer.py: error: the following arguments are required: --lr, --gradient_accumulation_steps, --num_train_epochs, --output_dir, --model_dir

**Comment by SKRohit** [comment]
optimal parameters are mentioned in the blog itself. If you cannot find some parameter then use the default value set in transformers library by huggingface, I had used those default values only.

**Comment by SumanthSrungavarapu** [comment]
> 
Thank you I. Already doing that.
My training is taking a very long time, can you suggest any free and powerful playforms other than colab( in colab you is given only for 6hrs) 
Or any way to skip the training?

**Comment by SKRohit** [comment]
I have already shared the model that I could find (check closed issues), and you could try kaggle as well for training.

**Comment by SumanthSrungavarapu** [comment]
> I have already shared the model that I could find (check closed issues), and you could try kaggle as well for training.


resolved

Here is the model and most probably the data file I used for training https://drive.google.com/file/d/1fmff72Z-9CvpHs2wwTBZJZYTKvbOxebk/view?usp=sharing and https://drive.google.com/file/d/1fmff72Z-9CvpHs2wwTBZJZYTKvbOxebk/view?usp=sharing. You can try them and see if they work as expected else you will have to train yourself. It has been some time I worked on this project so couldn't find all the data and models I used.

very sorry to disturb you , but you shared only the data link twice.
please provide the model file also, thank you
